frontend/isoko/src/components/BusinessPreview.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Rating } from 'react-simple-star-rating';
+import { Table } from 'react-bootstrap';
+import MinorityTag from './MinorityTag';
+import KeywordTag from './KeywordTag';
+
+const BusinessContainer = styled(Table)`
+   width: 85%;
+   height: 25%;
+   background-color: #fff;
+   border: 1px solid #cecece;
+   box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+   min-height: 20vh;
+   min-width: 60vh;
+   margin-bottom: 3%;
+`;
+
+const Photo = styled.img`
+   height: 100%;
```

Comment:
Similarly, we should use px for the photo's height and width. This will prevent the photo from being rectangular/weird on small or large devices. We might also need max/min width and heights as well.

<img width="301" alt="Screen Shot 2022-01-18 at 7 57 41 PM" src="https://user-images.githubusercontent.com/15805074/150061332-6221b6d9-725e-468c-8679-6d8cffa474c3.png">

Also add a very small border radius here and on the BusinessContainer component as on the mockups